instrumentation/jmx-metrics/library/src/test/java/io/opentelemetry/instrumentation/jmx/JmxTelemetryTest.java

```diff
@@ -27,20 +27,28 @@ void createDefault() {
   @Test
   void missingClasspathTarget() {
     JmxTelemetryBuilder builder = JmxTelemetry.builder(OpenTelemetry.noop());
-    assertThatThrownBy(() -> builder.addClassPathRules("should-not-exist"))
-        .isInstanceOf(IllegalArgumentException.class);
+    assertThatThrownBy(() -> builder.addClassPathResourceRules("should-not-exist"))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessageContaining("not found")
+        .hasMessageContaining("should-not-exist");
```

Comment:
[for reviewer] the wrong exception message was provided here due to the too broad `Exception` catch clause.